### DISCUSSION TITLE
TailwindCSS Dark Mode not working Facebook Customize Chat

### DISCUSSION THREAD

**Discussion by tuanductran** [tailwindlabs/tailwindcss#10423] [Help]
![Screen Shot 2023-01-13 at 21 22 10](https://user-images.githubusercontent.com/69758022/214996024-febf70aa-a8a7-495e-ac60-cd5d7426ff02.png)

**Comment by wongjn** [comment]
Is that something you have styled with Tailwind classes? Have you applied `dark:` variants to get its appearance as you'd like? Consider sharing the code for it so we can give some more practical solutions for you.
